experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -26,15 +27,38 @@ package_to_version () {
 
 # look up a binary with which and return the debian package it belongs to
 command_to_package () {
-    BINARY_PATH=$(readlink -f $(which $1))
+    local BINARY_PATH=$(readlink -f $(which $1))
```

Comment:
nit: per https://google.github.io/styleguide/shell.xml?showone=Variable_Names#Variable_Names, variable names should be lowercase.

Reply:
done, and elsewhere